docs/content/en/install/local/_index.md

```diff
@@ -1,441 +1,2316 @@
+demo-superuser@cqlsh:test> select * from users;
+
+ email          | name          | state
+----------------+---------------+-------
+ john@gamil.com |    John Smith |    NC
+  joe@gamil.com |     Joe Jones |    VA
+   sue@help.com |       Sue Sas |    CA
+    tom@yes.com | Tom and Jerry |    NV
+
+(4 rows)
+demo-superuser@cqlsh:test> exit
+cassandra@k8ssandra-1-worker2:/$ exit
+exit
+```
+
+Try another cqlsh operation on a different cluster.
+
+
+```bash
+kubectx kind-k8ssandra-3
+
+kubectl exec --stdin --tty demo-dc1-default-sts-0 -n k8ssandra-operator -- /bin/bash
+```
+
+**Output:**
+
+```bash
+Defaulted container "cassandra" out of: cassandra, server-system-logger, jmx-credentials (init), server-config-init (init)
+cassandra@k8ssandra-3-worker3:/$ cqlsh -u demo-superuser -p KT-ROFfbD-O9BzWS3Lxq
+Connected to demo at 127.0.0.1:9042
+[cqlsh 6.0.0 | Cassandra 4.0.1 | CQL spec 3.4.5 | Native protocol v5]
+Use HELP for help.
+demo-superuser@cqlsh> USE test;
+demo-superuser@cqlsh:test> SELECT * FROM users;
+
+ email          | name          | state
+----------------+---------------+-------
+ john@gamil.com |    John Smith |    NC
+  joe@gamil.com |     Joe Jones |    VA
+   sue@help.com |       Sue Sas |    CA
+    tom@yes.com | Tom and Jerry |    NV
+
+(4 rows)
+demo-superuser@cqlsh:test> exit
+cassandra@k8ssandra-3-worker3:/$ exit
+exit
+```
+
+Now try using the Stargate API. 
+
+**Output plus cqlsh &amp; stargate-service example:**
+
+```bash
+kubectx kind-k8ssandra-3
+
+kubectl exec --stdin --tty demo-dc3-default-sts-0 -n k8ssandra-operator -- /bin/bash
+
+Defaulted container "cassandra" out of: cassandra, server-system-logger, jmx-credentials (init), server-config-init (init)
+cassandra@k8ssandra-3-worker3:/$ cqlsh -u demo-superuser -p KT-ROFfbD-O9BzWS3Lxq demo-dc3-stargate-service
+Connected to demo at demo-dc3-stargate-service:9042
+[cqlsh 6.0.0 | Cassandra 4.0.1 | CQL spec 3.4.5 | Native protocol v4]
+Use HELP for help.
+demo-superuser@cqlsh> use test;
+demo-superuser@cqlsh:test> select * from users;
+
+ email          | name          | state
+----------------+---------------+-------
+ john@gamil.com |    John Smith |    NC
+  joe@gamil.com |     Joe Jones |    VA
+   sue@help.com |       Sue Sas |    CA
+    tom@yes.com | Tom and Jerry |    NV
+
+(4 rows)
+```
+
+## Helm
+You need to have [Helm v3+](https://helm.sh/docs/intro/install/) installed.
+
+Configure the K8ssandra Helm repository:
+
+```console
+helm repo add k8ssandra https://helm.k8ssandra.io/stable
+```
+
+Update your Helm repository cache:
+
+```console
+helm repo update
+```
+
+Verify that you see the `k8ssandra-operator` chart:
+
+```console
+helm search repo k8ssandra-operator
+```
+
+```console
+NAME                                CHART VERSION   APP VERSION DESCRIPTION
+k8ssandra/k8ssandra-operator        0.32.0          1.0.0       Kubernetes operator which handles the provision...
+```
+
+### Single Cluster 
```

Comment:
All the content from here down duplicates what's above... did you mean for this to be here still?

Reply:
Oh ok, I see what you mean.  In my next commit, I'll remove the whole ## helm section, but grab the first portion and insert it in the place that you noted in comments above.